cleanrl/td3_continuous_action.py

```diff
@@ -47,6 +47,8 @@ class Args:
     """total timesteps of the experiments"""
     learning_rate: float = 3e-4
     """the learning rate of the optimizer"""
+    num_envs: int = 2
```

Comment:
I would default this to 1

Reply:
Sure. Probably also worth discussing how to handle total_timesteps with multiple environments.

Reply:
@pseudo-rnd-thoughts It seems like in sb3 they do it like this:
Lets say total_timesteps is 100_000. 
Then they actually run 100_000 * num_envs steps, because for each timestep num_envs step are executed.